Core/Core/Profile/Help/Model/HelpLinkRoute.swift

```diff
@@ -20,7 +20,11 @@ extension HelpLink {
     var route: (path: String, options: RouteOptions)? {
         switch id {
         case "instructor_question":
-            return ("/conversations/compose?instructorQuestion=1&canAddRecipients=", .modal(.formSheet, embedInNav: true))
+            let autoTeacherSelectKey = ComposeMessageOptions.QueryParameterKey.autoTeacherSelectKey.rawValue
+            let recipientsDisabledKey = ComposeMessageOptions.QueryParameterKey.recipeientsDisabledKey.rawValue
+            let alwaysShowRecipientsKey = ComposeMessageOptions.QueryParameterKey.alwaysShowRecipientsKey.rawValue
+            let trueValue = String(true)
```

Comment:
Is there a reason for not simply using `"true"` directly?

Reply:
I think it's just more understandable then a simple string value